### PR TITLE
fix: removed empty files and translated Chinese to English

### PR DESCRIPTION
**Pull request by XiaoFei9704**
#### Description

Fix: OpenScenario2.0: removed empty files and translated Chinese to English

#### Where has this been tested?

  * **Platform(s):** Ubuntu 20.04
  * **Python version(s):** 3.8
  * **Unreal Engine version(s):** 4.26
  * **CARLA version:** 0.9.15

- - -
This change is [Reviewable](https://reviewable.io/reviews/carla-simulator/scenario_runner/1063)

### CODE REVIEW

**Review by PabloVD — APPROVED** [review]

**Review by PabloVD — APPROVED** [review]
> Reviewed 33 of 33 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/scenario_runner/1063)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @XiaoFei9704)

File: srunner/scenarioconfigs/osc2_scenario_configuration.py [hunk 1]
```diff
@@ -17,11 +17,16 @@
 from srunner.osc2_dm.physical_object import *
 from srunner.osc2_dm.physical_types import Physical, Range
 
-# 标准库
 from srunner.osc2_stdlib.path import Path
 
+<<<<<<< HEAD
```
Comment (kristjard) [inline srunner/scenarioconfigs/osc2_scenario_configuration.py:22]: Pretty sure this isn't supposed to be merged :)

Comment (PabloVD) [reply srunner/scenarioconfigs/osc2_scenario_configuration.py:22]: Fixed, thanks for noticing it.

**Comment by hsiang-lee** [comment]
https://osc2studio.online

**Pull request merged**